homework_7_261122/src/cars/Car.java

```diff
@@ -0,0 +1,76 @@
+package cars;
+
+public class Car {
+    int maxSpeed;
+    int fuelAmount;
+    int productionYear;
+    double price;
+    String brand;
+    int currentSpeed;
+    boolean isEngineStarted = false;
+
+    public Car(int maxSpeed, int fuelAmount, int productionYear, double price, String brand) {
+        this.maxSpeed = maxSpeed;
+        this.fuelAmount = fuelAmount;
+        this.productionYear = productionYear;
+        this.price = price;
+        this.brand = brand;
+    }
+
+    public void getMaxSpeed() {
+        System.out.println("Max speed: " + maxSpeed);
```

Comment:
you need to return a variable not to print it, it relates to all get methods

Reply:
Fixed.